operate/client/e2e-playwright/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+const URL_PATTERN = /^.*\/(api|v2).*$/i;
```

Comment:
🔧 To make clear this is about api urls?
```suggestion
const URL_API_PATTERN = /^.*\/(api|v2).*$/i;
```